### PR TITLE
Expose original browser events for igxChip and igxChipsArea

### LINKED ISSUE #2487: The igxChip onSelection doesn't expose the original browser event
### Description  
The `igxChip` `onSelection` output should expose the original browser event, if there's such.

 * igniteui-angular version: 6.1.3
 * browser: N/A

### PR DESCRIPTION
**Pull request by skrustev**
Closes #2487

### CODE REVIEW

File: projects/igniteui-angular/src/lib/chips/chip.component.ts [hunk 1]
```diff
@@ -457,8 +433,9 @@ export class IgxChipComponent implements AfterViewInit {
     /**
      * @hidden
      */
-    public onChipRemove() {
+    public onChipRemove(event) {
```
Comment (mpavlinov) [inline projects/igniteui-angular/src/lib/chips/chip.component.ts]: @skrustev Please add the event type.

File: projects/igniteui-angular/src/lib/chips/chip.component.ts [hunk 2]
```diff
@@ -474,20 +451,47 @@ export class IgxChipComponent implements AfterViewInit {
     /**
      * @hidden
      */
-    public onChipRemoveEnd() {
+    public onChipRemoveEnd(event) {
```
Comment (mpavlinov) [inline projects/igniteui-angular/src/lib/chips/chip.component.ts]: @skrustev Please add the event type.

File: projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts [hunk 3]
```diff
@@ -479,6 +482,7 @@ export class IgxDragDirective implements OnInit, OnDestroy {
     public onPointerMove(event) {
```
Comment (mpavlinov) [inline projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts]: @skrustev Please add the event type.

Comment (skrustev) [reply projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts]: I am using different approaches to getting the pageX/pageY for the different types of interaction, and specifying the types would result in errors from the TypeScript parser.

**Pull request merged**